scripts/search.py

```diff
@@ -172,17 +171,41 @@ def create_blast_graph(forward_blast_high_scoring_pairs, reverse_blast_high_scor
     :param reverse_blast_high_scoring_pairs: The reverse blast high scoring pairs.
     :return: A graph representation of forward and reverse HSPs.
     """
-    blast_graph = Graph()  # Creates graph to map BLAST hits
+    blast_graph = nx.Graph()  # Creates graph to map BLAST hits
```

Comment:
Construction of the graph via the new library (networkx)

Reply:
The forward and reverse bitscores are both retained as edge attributes. Currently, the `search.py` code does not use the forward bitscores for any of the logic, but I thought they might come handy in the future if the code is ever extended to do more complex graph manipulations.